### DISCUSSION TITLE
Question not issue regarding IdentifiedArray

### DISCUSSION THREAD

**Discussion by azamsharp** [pointfreeco/swift-identified-collections#45] [Q&A]
In the implementation of IdentifiedArray what makes it possible for it to be initialized as an array like the code shown below: 

@Published var tasks2: IdentifiedArray<CKRecord.ID, TaskItem> = []

**Comment by stephencelis** [comment]
Swift has an `ExpressibleByArrayLiteral` protocol that types can conform to in order for array literal syntax to work. Our conformance is here:

https://github.com/pointfreeco/swift-identified-collections/blob/a08887de589e3829d488e0b4b707b2ca804b1060/Sources/IdentifiedCollections/IdentifiedArray/IdentifiedArray%2BExpressibleByArrayLiteral.swift

**Comment by azamsharp** [comment]
Thanks Sent from my iPhoneOn Dec 30, 2022, at 2:08 PM, Stephen Celis ***@***.***> wrote:﻿
Swift has an ExpressibleByArrayLiteral protocol that types can conform to in order for array literal syntax to work. Our conformance is here:
https://github.com/pointfreeco/swift-identified-collections/blob/a08887de589e3829d488e0b4b707b2ca804b1060/Sources/IdentifiedCollections/IdentifiedArray/IdentifiedArray%2BExpressibleByArrayLiteral.swift
